plugins/woocommerce/client/legacy/css/twenty-twenty-one.scss

```
@@ -3050,6 +3050,20 @@ a.reset_variations {
 	}
 }
 
+@media only screen and (max-width: 768px) {
+	.woocommerce-message,
+	.woocommerce-error li,
+	.woocommerce-info {
+		padding: 1rem 1.5rem;
+
+		.button {
+			margin-left: 10px;
+			min-width: 100px;
+			padding: 10px 15px !important;
```

If you agree with the suggestion above, I think we can also do the same here
```suggestion
			padding: $gap-small $gap !important;
```

I'm wondering if we can also get rid of the `!important` statement here, could you share why it was needed to add the `!important` here? Maybe I can help with finding another way 😄

---

Thanks for the suggestion, didn't even think of looking for what global variables Twenty Twenty One uses. And I initially had `!important` here to override styles Twenty Twenty One adds itself but in following your other suggestion, that is no longer needed. Thanks!